### ISSUE TITLE
Question: Will simulations work if no previous history?

### ISSUE THREAD

**Issue by majo551** [slashback100/presence_simulation#108]
I just came across this fantastic integration. Reading through the documentation, is my understanding correct that if I just recently added  light controllers to the HA and they have no history yet, or when I am longer away and no lights used and I start the simulator after , there will be no simulation  running on these lights due to the no changes /history to the  grouped lights?
Also there is a light in one room not used frequently but when away, it would be great to have it randomly turned on/off regardless of my last week usage. 
 In other words, if the light in the group used for simulations has no history/usage, there would be a default random  turning on/off set when presence simulator switched on.
Thanks, 
Marian

**Comment by slashback100** [comment]
Yes, your understanding is correct. 
If no historic, nothing will happen. 

Using random switch if nothing happens is not the philosophy of this integration, I guess other integrations exists to have that behavior. 

Sorry :-)
